open-machine-learning-jupyter-book/assignments/ml-advanced/ensemble-learning/random-forests-intro-and-regression.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">%matplotlib inline
</pre><p> </p><p>put this line to the first</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
done